test/sse_test.dart

```diff
@@ -206,37 +206,17 @@ void main() {
 
       // Close the underlying connection.
       closeSink(connection);
-
-      // The isInKeepAlivePeriod flag may only be set for a short period because
-      // the client may connect very quickly, so only pump until it changes.
-      var maxPumps = 100;
-      while (!connection.isInKeepAlivePeriod && maxPumps-- > 0) {
-        await pumpEventQueue(times: 1);
-      }
-
-      // Ensure there's still a connection and it's marked as in the keep-alive
-      // state.
-      expect(connection.isInKeepAlivePeriod, isTrue);
```

Comment:
@grouma - Do you think that the round tripping of the value `'bar'` through the same `connection` object as was used before and already closed should be sufficient for this test?

I could add a stream of reconnect events, but I don't think it's necessary to demonstrate correctness here.
WDYT?

Reply:
That's probably fine since `closeSink` is synchronous and should guarantee that the connection is severed before we send `bar`.